.github/workflows/main.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
           python-version: 3.10.4
 
       - name: Install Requirements
-        run: >-
+        run: |
```

Comment:
Interesting, not sure why CI did not fail earlier if this was an issue.

Reply:
oh man. nice catch